Automattic/wp-calypso
# Update storage for Woo Express Essential to 50GB

## Proposed Changes

* This PR updates the feature list for Woo Express Essential to include 50GB of storage

## Testing Instructions

* Open up the Calypso.live branch, and for a site on any of the following plans, navigate to _Upgrades_ -> _Plans_:
  - eCommerce trial
  - Woo Express Essential
  - Woo Express Performance
* Verify that the Woo Express Essential plan is listed as having 50GB of storage

### Screenshots

#### Before

<img width="994" alt="Screenshot 2023-04-06 at 16 00 59" src="https://user-images.githubusercontent.com/3376401/230401275-c3bc697f-3afa-44cb-9f13-b5751e036d8b.png">

#### After
<img width="994" alt="Screenshot 2023-04-06 at 15 59 58" src="https://user-images.githubusercontent.com/3376401/230401245-a1d359a5-0ccc-4750-bb2f-c2d5f85c54db.png">


## Pre-merge Checklist

- [N/A] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [N/A] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [N/A] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [N/A] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?
